src/components/dataset/schemaCreation/DatasetSchemaBuilderView.tsx

```diff
@@ -0,0 +1,953 @@
+                      return isFrom || isTo;
+                    })
+                    .map((rel: RelationshipModel) => ({
+                      tooltip: rel.name,
+                      icon: (
+                        <IconButton
+                          size="small"
+                          color="primary"
+                          className={classes.relationshipButton}
+                          onMouseEnter={() => highlightRelationshipColumns(rel)}
+                          onMouseLeave={() => setOutlinedRelationships([])}
+                          onClick={() => openRelationshipEditor({ rel })}
+                          disableRipple
+                        >
+                          <i className="fa fa-link-horizontal" />
+                        </IconButton>
+                      ),
+                    })) || []
+                }
+              />
+            </div>
+          </div>
+          {selectedTable !== -1 &&
+            selectedColumn === -1 &&
+            datasetSchema.tables[selectedTable] &&
+            renderTableDetails()}
+          {selectedTable !== -1 &&
+            selectedColumn !== -1 &&
+            datasetSchema.tables[selectedTable]?.columns[selectedColumn] &&
+            renderColumnDetails()}
+        </div>
+
+        <div>
+          <Typography variant="h4" className={classes.section}>
+            JSON view
+          </Typography>
+          <CodeMirror
+            data-cy={JSON.stringify(datasetSchema)}
+            value={_.keys(datasetSchema).length > 0 ? JSON.stringify(datasetSchema, null, 2) : ''}
+            height="400px"
+            theme={okaidia}
+            extensions={[javascript({ jsx: true, typescript: true })]}
+            onChange={onJsonViewerChange}
+            className={classes.jsonWrapper}
+            placeholder="If you already have json, please paste your code here"
+          />
+        </div>
+      </div>
+
+      {relationshipModalOpen && (
+        <DatasetSchemaRelationshipModal
+          datasetSchema={datasetSchema}
+          isEditMode={relationshipModalDefaultValues.isEditMode}
+          defaultRelationshipName={relationshipModalDefaultValues.name}
+          defaultRelationshipFrom={relationshipModalDefaultValues.from}
+          defaultRelationshipTo={relationshipModalDefaultValues.to}
+          defaultExpandedTablesFrom={relationshipModalDefaultValues.expandedTablesFrom}
+          defaultExpandedTablesTo={relationshipModalDefaultValues.expandedTablesTo}
+          onSubmit={(data: RelationshipModel) => {
+            if (relationshipModalDefaultValues.isEditMode) {
+              editRelationship(data);
+            } else {
+              createRelationship(data);
+            }
+          }}
+          onClose={() => {
+            setRelationshipModalOpen(false);
+          }}
+          onDelete={deleteRelationship}
+        />
+      )}
+    </div>
+  );
+});
+
```

Comment:
I noticed that we can set relationships in this schema wizard, but we can't set assets.  AFAIK, a dataset must have assets in order to be used in TDR UI's snapshot creation wizard.  I'd recommend that we prioritize supporting their specification here.